product/roundhouse.tasks/Roundhouse.cs

```diff
@@ -110,9 +112,15 @@ bool ITask.Execute()
         public string ScriptsRunErrorsTableName { get; set; }
 
         [Obsolete("Use EnvironmentNames")]
-        public string EnvironmentName { get; set; }
-
-        public string EnvironmentNames { get; set; }
+        public string EnvironmentName {
+            get { return EnvironmentNames.SingleOrDefault(); }

```

Comment:
This will crash if you have more than one environment in the list. That's probably OK? We don't have a better way of solving it, do we?

Reply:
I would say the exception is the desired behavior. Using a mixture of EnvironmentName and EnvironmentNames is incorrect. I like to fail loudly when being mis-used.

Reply:
I agree. Leave it as is :)